validator/src/main.rs

```diff
@@ -1853,6 +1854,7 @@ pub fn main() {
         bind_ip_addr: bind_address,
         public_tpu_addr,
         public_tpu_forwards_addr,
+        num_tvu_sockets: tvu_receive_threads,
```

Comment:
thoughts on trying to keep the names consistent here? `sockets` vs. `threads`?

Reply:
Good question, so here is my thinking:
- In the context of `Node`, `Node` is aware of how many sockets it creates.
- In the context of `Validator`, we later create one thread per socket (here are some pieces of the callstack)
https://github.com/anza-xyz/agave/blob/f4307ad2d33317056114d847eb8b2243ad082e7e/core/src/validator.rs#L1284-L1287
https://github.com/anza-xyz/agave/blob/f4307ad2d33317056114d847eb8b2243ad082e7e/core/src/tvu.rs#L177-L178
https://github.com/anza-xyz/agave/blob/f4307ad2d33317056114d847eb8b2243ad082e7e/core/src/shred_fetch_stage.rs#L164-L168

So, we have a 1-to-1 mapping between num threads and num sockets. At that point, it is a tossup in my opinion on whether to use threads or sockets at the CLI level. I opted for threads given that all of the other args I've added like this use `threads`. I did add a note about sockets in the CLI help tho:
> "Number of threads (and sockets) to use for receiving shreds on the TVU port

Reply:
i think that is a fair argument! and ya i agree i think threads is better from a validator cli perspective. and having threads in `NodeConfig` is not great for how its implemented already